app/controllers/users_controller.rb

```diff
@@ -9,4 +9,7 @@ def update
   def user_params
     params.require(:user).permit(:avatar)
   end
+  def search
+    @user = User.search(params[:search])
```

Comment:
what is this search for?

Reply:
remember the search bar on the header is ment for finding products, not users

Reply:
That's something I forgot to delete. Should I delete it and push it again?

Reply:
yes! it's better to avoid merging unnecessary code to develop